onnx/defs/reduction/defs.cc

```diff
@@ -19,7 +19,7 @@ False instead of True.)DOC";
     schema.SetDoc(doc);
     schema.Attr(
         "axes",
-        "A list of integers, along which to reduce.",
+        "A list of integers, along which to reduce. If not specified, reduce operation will be along the last axis.",
```

Comment:
We should align with numpy, if no axes are specified, we should do the reduction along all the dimensions.

Reply:
resolved